### ISSUE TITLE
Luaerror in MetalFeatures GL4

### ISSUE THREAD

**Issue by Arch-Shaman** [ZeroK-RTS/Zero-K#4678]
[infolog.txt](https://github.com/ZeroK-RTS/Zero-K/files/8515981/infolog.txt)

![image](https://user-images.githubusercontent.com/48730089/164080837-6c9ca17a-6e05-4ee8-b358-589b86126355.png)

Occurs in http://zero-k.info/Battles/Detail/1348062 (guardian commander in middle of map around 10 minutes in?)

**Comment by GoogleFrog** [comment]
Does this reproduce for you?

**Comment by Arch-Shaman** [comment]
Yes, with view selected player (in spectating options) enabled. ~13:46.

**Comment by GoogleFrog** [comment]
The commander is selected there so it looks like you've done extra things. Tell me _exactly_ what you did. This could entail not doing anything that is too hard to describe (such as using the scroll wheel),

Actually, you could override `gui_metal_features_gl4.lua` yourself and put FeatureEchos in likely locations. One before `WG.HighlightUnitGL4` seems good.

**Comment by Arch-Shaman** [comment]
I selected the commander, and just let it do its thing (so as to make it update). I'll put down some echos locally.
